Material_Button/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@ohos/material_button",
```

Comment:
change the name to material-button or materialbutton

Reply:
name changed.